src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+
+import EnterEmailAdress from '../../app/assets/design-patterns/user-authentication/enter-email-address.png';
+import AcceptEULA from '../../app/assets/design-patterns/user-authentication/accept-eula.png';
+import VerifyEmail from '../../app/assets/design-patterns/user-authentication/verify-email.png';
+import CreatePassword from '../../app/assets/design-patterns/user-authentication/create-password.png';
+import AccountDetails from '../../app/assets/design-patterns/user-authentication/account-details.png';
+import WelcomeOnboard from '../../app/assets/design-patterns/user-authentication/welcome-onboard.png';
+import EnterOrganizationCode from '../../app/assets/design-patterns/user-authentication/enter-organization-code.png';
+import EnterOrganizationPasscode from '../../app/assets/design-patterns/user-authentication/enter-organization-passcode.png';
+import EnterOrganizationConfirm from '../../app/assets/design-patterns/user-authentication/enter-organization-confirm.png';
+import EnterOrganizationWelcome from '../../app/assets/design-patterns/user-authentication/enter-organization-welcome.png';
+import Login from '../../app/assets/design-patterns/user-authentication/login.png';
+import ForgotPasswordEnterEmail from '../../app/assets/design-patterns/user-authentication/forgot-password-email.png';
+import ForgotPasswordConfirmSent from '../../app/assets/design-patterns/user-authentication/forgot-password-confirm.png';
+import ForgotPasswordNewPassword from '../../app/assets/design-patterns/user-authentication/forgot-password-password.png';
+import ForgotPasswordWelcome from '../../app/assets/design-patterns/user-authentication/forgot-password-welcome.png';
+import ChangePassword from '../../app/assets/design-patterns/user-authentication/change-password.png';
+import ChangePasswordWelcome from '../../app/assets/design-patterns/user-authentication/change-password-confirm.png';
+
+<!-- Behaviors -->
+
+import EmailCommunicationInvitation from '../../app/assets/design-patterns/user-authentication/email-communication-invitation.png';
+import EmailCommunicationVerification from '../../app/assets/design-patterns/user-authentication/email-communication-verification.png';
+import ErrorStates from '../../app/assets/design-patterns/user-authentication/error-states.png';
+
+<!-- KEYWORDS: design pattern login registration register password forget forgot account authentication auth join -->
+
+# User Authentication
+
+<ImageGrid images={[LoginBanner]} regularWidth />
+
+Most applications will require some form of user authentication to ensure that only authorized users have access.
+
+<!-- @TODO switch login links over to the "Component Recommendations style" instead of an "Interactive Code example" once we have one designed. -->
+
+<React.Fragment>
+    <DemoCard repository={'login'} angular react ionic reactNative />
+    <Divider />
+</React.Fragment>
+
+# Variations
+
+<ImageGrid
+    images={[ViewPortLoginImage, ViewPortOverlayImage]}
```

Comment:
I try to use this paragraph to explain how they would look differently on mobile & desktop, and then I can present only the mobile screens from here on.

Reply:
Moved to the behavior section under the headline "Responsive Design"

Reply:
The purpose of those screens was to show (1) that we have several different auth-related workflows and (2) how they work on different viewport.

Reply:
I get that. I was just a little confused because it's the first image under variations, but it's only showing 2 of the (6?).

If you leave it in, you should update the caption — it refers to all 4 as login screens.